[IMP] rental: change to physical service product (#18943)
odoo/documentation: content/applications/sales/rental/configure_products/service_products.rst

================================
Physical service rental products
================================

There are two types of service products in the **Rental** app that require different configurations:
physical and non-physical (labor). This document will refer to non-physical services as labor going

> forward.

Reviewer: This intro is a bit confusing, it makes it hard to understand exactly what will be included in the doc. 
From my take away, the doc should cover *both* types of service products, in which case I wouldn't think the doc title needs to specify "physical" (especially since there is already a doc titled "physical products" I realize this one is "physical service" but it is a bit confusing)
Also, you mention using the term "larbor" going forward, but then it doesn't appear again in the text.

Author: I am making  separate page for the Labor services because having it in one page makes the document super long.

Settings`.

The following configurations assume the **Rental**, **Planning**, and **Sales** apps are installed.

Configure materials and a role for the physical product
=======================================================

Before creating a service product in the **Rental** app, configure :ref:`materials
<planning/materials>` and :ref:`create a role <planning/roles>` in the **Planning** app for the
physical rental products (such as storage units or conference rooms).

This allows the **Planning** app to create and assign shifts for the physical rental product, and
the *Role* links the rental materials to the rental service. Whenever the service is added to a
rental order, the **Rental** app syncs with the **Planning** app to check and update the material
shift availability.

.. example::
   Shady Grove needs to add a new room tier to the **Rental** app, the *Deluxe Room*. There are
   seven rooms that belong in this tier. Create a new *Role* by navigating to the
   :menuselection:`Planning app --> Configuration --> Roles`. Click :guilabel:`New` and create the
   :guilabel:`Deluxe Room Role`.

>    .. tip::

Reviewer: I would move this above the example box, having a box with images and another box is a bit cluttered

   To add the seven rooms as a resource for the *Role*, navigate to :menuselection:`Configuration
   --> Materials`. Click :guilabel:`New`, enter the room number, and :guilabel:`Deluxe Room` for the
   :guilabel:`Role` column. Repeat that for all seven rooms.

   .. image:: service_products/example-planning-materials.png
      :alt: Example of configured materials for the Deluxe Room.

   Create the :guilabel:`Deluxe Room` as a new *Service* product type by navigating to
   :menuselection:`Rental app --> Products`. Click :guilabel:`New`, then configure the Deluxe Room
   as a :guilabel:`Service` type product with the :guilabel:`Plan Services` checkbox enabled and the
   :guilabel:`Deluxe Room` role assigned.

      .. image:: service_products/plan-services-field.png
         :alt: Example of the Plan Serviced field configured to the Deluxe Room role.

.. _rental/service_products/new-service:

Create a new service product
============================

To set up a new rental service, go to the :menuselection:`Rental app --> Products --> Products` and
then click :guilabel:`New`.

.. important::
   The **Sales** and **Planning** apps must be installed for the *Prepaid/Fixed Price*

>    option of the *Invoicing Policy* and *Plan Services* fields to be available.

Reviewer: ```suggestion
   The **Sales** and **Planning** apps must be installed for the *Prepaid/Fixed Price* option of the *Invoicing Policy* and *Plan Services* fields to be available.
```
Double check line lengths

.. image:: service_products/physical-service-product-form.png
   :alt: Example of a configured physical service product in the Rental app.

Click the :icon:`oi-arrow-right` :guilabel:`(Internal link)` icon to open the product’s *Role* page.
Enable the :guilabel:`Sync Shifts and Rental Orders` checkbox.

.. image:: service_products/rental-role-page.png
   :alt: Example of a configured Role page for the Deluxe Room in the Rental app.

.. tip::
   Assign a :guilabel:`Category` for room booking. It separates rooms from other services and can be
   used for reports on room occupancy.

.. _rental/service_products/rental-periods-pricing:

Set a base rental period and price
----------------------------------

Set the base rental price by entering the lowest rental price in the :guilabel:`Sales Price` field.
Next, click the :guilabel:`Sales` tab, then configure the following *Rental* section fields where
applicable:

- :guilabel:`Periodicity`: The unit of time the product will use for rental prices.
- :guilabel:`Padding Time`: Blocks a rental product from being available for reservations. The
  setting is set to an hourly unit. This is available only if :guilabel:`Hours` is selected in

>   the :guilabel:`Periodicity` field and the **Inventory app** is installed.

Reviewer: ```suggestion
  setting is set to an hourly unit. This is available only if :guilabel:`Hours` is selected in the :guilabel:`Periodicity` field and the **Inventory app** is installed.
```
Double check line lengths

- :guilabel:`Pickup`: The earliest time the customer can pick up the product to begin the rental
  period.
- :guilabel:`Return:` The latest time the customer can return the product.

Set up variants
---------------

.. important::

>    The *Variant* feature must be enabled for this tab to display.

Reviewer: ```suggestion
   The *Variants* feature must be enabled for this tab to display.
```

================================
Physical service rental products
================================

There are two types of service products in the **Rental** app that require different configurations:
physical and non-physical (labor). This document focuses on the configuration of physical rental

> service products. A physical rental service product is a physical product that doesn’t require any

Reviewer 2: ```suggestion
service products. A physical rental service product is a physical product that doesn't require any
```

Changing from curly apostrophe to straight apostrophe. This typically happens when copying from a rich-text-based editor to VS Code.

Not sure if you're aware, but VS Code actually highlights these as you draft. I would look out for those apostrophes and change them as you see them.

<img width="86" height="50" alt="Image" src="https://github.com/user-attachments/assets/3b3591b1-72d5-477c-822e-dcfea91c12f1" />

stock movements. Some examples are:

- Hotel rooms
- Conference rooms
- Work stations
- Storage units

.. _rental/service_products/settings:

Settings
========

Depending on the type of service product, the requirements differ. To learn more about the default
settings for rental products, refer to the :ref:`Configuration <rental/product_type/configuration>`
section on the Rental product types page.

> To access the Rental app’s settings, navigate to :menuselection:`Rental app --> Configuration -->

Reviewer 2: ```suggestion
To access the Rental app's settings, navigate to :menuselection:`Rental app --> Configuration -->
```

Same apostrophe issue.

Settings`.

The following configurations assume the **Rental**, **Planning**, and **Sales** apps are installed.

Configure materials and a role for the physical product
=======================================================

Before creating a service product in the **Rental** app, configure :ref:`materials
<planning/materials>` and :ref:`create a role <planning/roles>` in the **Planning** app for the
physical rental products (such as storage units or conference rooms).

This allows the **Planning** app to create and assign shifts for the physical rental product, and
the *Role* links the rental materials to the rental service. Whenever the service is added to a
rental order, the **Rental** app syncs with the **Planning** app to check and update the material
shift availability.

   .. tip::

>       It’s recommended to name the role after the service.

Reviewer 2: ```suggestion
.. tip::
   It's recommended to name the role after the service.
```

Reducing the indent and fixing the rich-text apostrophe issue.

.. example::
   Shady Grove needs to add a new room tier to the **Rental** app, the *Deluxe Room*. There are
   seven rooms that belong in this tier. Create a new *Role* by navigating to the
   :menuselection:`Planning app --> Configuration --> Roles`. Click :guilabel:`New` and create the
   :guilabel:`Deluxe Room Role`.

   .. image:: service_products/example-planning-role.png
      :alt: Example of configured role for the Deluxe Room.

   To add the seven rooms as a resource for the *Role*, navigate to :menuselection:`Configuration
   --> Materials`. Click :guilabel:`New`, enter the room number, and :guilabel:`Deluxe Room` for the

>    :guilabel:`Role` column. Repeat that for all seven rooms.

Reviewer 2: ```suggestion
   :guilabel:`Role` column. Repeat that process for all seven rooms.
```

Making the sentence slightly clearer.

   .. image:: service_products/example-planning-materials.png
      :alt: Example of configured materials for the Deluxe Room.

   Create the :guilabel:`Deluxe Room` as a new *Service* product type by navigating to
   :menuselection:`Rental app --> Products`. Click :guilabel:`New`, then configure the Deluxe Room
   as a :guilabel:`Service` type product with the :guilabel:`Plan Services` checkbox enabled and the
   :guilabel:`Deluxe Room` role assigned.

      .. image:: service_products/plan-services-field.png

>          :alt: Example of the Plan Serviced field configured to the Deluxe Room role.

Reviewer 2: ```suggestion
   .. image:: service_products/plan-services-field.png
      :alt: Example of the Plan Services field configured to the Deluxe Room role.
```

Fixing indent and the name of the field.

.. _rental/service_products/new-service:

Create a new service product
============================

To set up a new rental service, go to the :menuselection:`Rental app --> Products --> Products` and
then click :guilabel:`New`. The new product form displays with the :guilabel:`General Information`
tab open as default.

Initial product configuration
-----------------------------

.. important::
   The **Sales** and **Planning** apps must be installed for the *Prepaid/Fixed Price* option of the
   *Invoicing Policy* and *Plan Services* fields to be available. Enabling the *Sales* checkbox
   displays the *Invoicing Policy*.

In the new product window, the :guilabel:`Sales` checkbox is already enabled by default. Select the
:guilabel:`Product Type` as :guilabel:`Service`. In the :guilabel:`Invoicing Policy` drop-down menu,
select :guilabel:`Prepaid/Fixed Price`. Enable the :guilabel:`Plan Services` checkbox and either
:ref:`create a new role <planning/roles>` or select a pre-existing one.

.. image:: service_products/physical-service-product-form.png
   :alt: Example of a configured physical service product in the Rental app.

> Click the :icon:`oi-arrow-right` :guilabel:`(Internal link)` icon to open the product’s *Role* page.

Reviewer 2: ```suggestion
Click the :icon:`oi-arrow-right` :guilabel:`(Internal link)` icon to open the product's *Role* page.
```

Same apostrophe issue.

Enable the :guilabel:`Sync Shifts and Rental Orders` checkbox.

.. image:: service_products/rental-role-page.png
   :alt: Example of a configured Role page for the Deluxe Room in the Rental app.

.. tip::
   Assign a :guilabel:`Category` for room booking. It separates rooms from other services and can be
   used for reports on room occupancy.

.. _rental/service_products/rental-periods-pricing:

Set a base rental period and price
----------------------------------

To set the base rental price for a rental product, open the *General Information* tab and enter the
lowest rental price in the :guilabel:`Sales Price` field. Next, click the :guilabel:`Sales` tab,
then configure the following *Rental* section fields where applicable:

- :guilabel:`Periodicity`: The unit of time the product will use for rental prices.
- :guilabel:`Padding Time`: Blocks a rental product from being available for reservations. The
  setting is set to an hourly unit. This is available only if :guilabel:`Hours` is selected in the
  :guilabel:`Periodicity` field and the **Inventory app** is installed.
- :guilabel:`Pickup`: The earliest time the customer can pick up the product to begin the rental
  period.

> - :guilabel:`Return:` The latest time the customer can return the product.

Reviewer 2: ```suggestion
- :guilabel:`Return`: The latest time the customer can return the product.
```

Seems the colon was misplaced.

Optional: specify rental variants
---------------------------------

.. important::
   The *Variants* feature must be enabled for this tab to display.

Click :guilabel:`Add a line`, then select any room amenity or room item from the
:guilabel:`Attribute` drop-down menu. To create a new one, enter the name and click
:guilabel:`Create and edit…` to configure the attribute and values.

.. example::
   Shady Grove has a list of amenities it wants to configure for their Deluxe Room. The room has two
   types of bed configurations: two double beds or one king bed. The maximum occupancy per room is
   two adults or two adults and a child.

   Navigate to the :menuselection:`Rental app --> Products` and click :guilabel:`New` to create a
   new product. On the :guilabel:`Attributes & Variants` tab, click :guilabel:`Add a line` and
   select the :guilabel:`Bed` as an :guilabel:`Attribute`. In the :guilabel:`Values` column, add
   :guilabel:`2 Double` and :guilabel:`1 King`. Repeat these steps until all amenities have been
   configured.

   .. image:: service_products/example-room-variants.png

>      :alt: Example of the Attribute & Variants tab configured for a hotel room.

Reviewer 2: ```suggestion
      :alt: Example of the Attribute & Variants tab configured for a hotel room.
```

Indenting the alt tag appropriately.

> .. _rental/service_products/ additional-pricing:

Reviewer 2: ```suggestion
.. _rental/service_products/additional-pricing:
```

Stray space.

.. important::
   The :ref:`Pricelists <sales/products/pricelist-configuration>` feature must be enabled for this
   tab to display.

Rental rates can also be configured as a new price rule for an existing pricelist using the
:guilabel:`Prices` tab on the product form. If no pricelist is configured beforehand, the *Default*
pricelist is selected.

It is recommended to create a new pricelist first instead of using the *Default* pricelist. Keeping
the *Default* pricelist blank ensures there is a clean pricelist for the base rental rate.

Navigate to :menuselection:`Products --> Products`, then click the desired product. Click the
:guilabel:`Prices` tab and click :guilabel:`Add a price`.

Select the desired :guilabel:`Pricelist`, then enter the minimum time required for the price change
to trigger in the :guilabel:`Min. Quantity` column. The :guilabel:`Min. Quantity` column is based on
the *Periodicity* field in the :guilabel:`Sales` tab.

Lastly, enter the :guilabel:`Price` rate. Click the :icon:`fa-cloud-upload` :guilabel:`(Save
manually)` icon near the top to save.

.. tip::
   Add a date range in with the :guilabel:`Validity` column. To add a :guilabel:`Validity` column,

>    click the :icon:`oi-settings-adjust` :guilabel:`(Settings)` icon and enable :guilabel:`Validity`

Reviewer 2: ```suggestion
   Add a date range in the :guilabel:`Validity` column. To add a :guilabel:`Validity` column, click
   the :icon:`oi-settings-adjust` :guilabel:`(Settings)` icon and enable :guilabel:`Validity`.
```

Stray preposition (with) and missing a period.

   .. image:: service_products/example-prices-tab.png
      :alt: Sample of the Prices tab of service product in the Rental app.

eCommerce features
------------------

.. important::
   This tab is only available if the :guilabel:`eCommerce` module is installed.

The :guilabel:`eCommerce` tab configures the product page on the website. Refer to the :ref:`Product
visibility <ecommerce/products/publish-products>` and :ref:`Product configuration
<ecommerce/products/product-configuration>` sections for the **eCommerce** module for configuration
instructions.

Any selected days in the *Unavailability days* section in the :ref:`Rental app's settings
<rental/product_type/configuration>` are only applied to online booking. If the product isn't

> published to the website then the setting does not go into affect.

Reviewer 2: ```suggestion
published to the website then the setting does not go into effect.
```

   .. image:: service_products/return-form-example-product-service.png
      :alt: Sample of a Validate a return form with a rental product and service listed.

.. seealso::
   - :doc:`../../../services/planning`
   - :doc:`../../sales/products_prices/prices/pricing`

> 

Reviewer 2: You can delete this extra line at the end of the file -- only one blank line is required.
